packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -6,22 +6,19 @@ excerpt: >-
 
 # Client-based rate limits
 
-Client-based limiting uses either of these two endpoints:
+Client-based rate limiting applies to unauthenticated endpoints within Okta that are used during the client application access process. This process can include multiple API endpoints that differ based on your type of org: Okta Identity Engine or Okta Classic Engine.
 
-* OAuth 2.0 `/authorize`
-* `/login/login.htm`
+The Classic Engine and the Identity Engine both include rate-limiting for the OAuth 2.0 API endpoint `/authorize`. This endpoint uses a combination of the Client ID, user's IP address, and Okta device identifier to provide granular isolation for requests made to the OAuth 2.0 `/authorize` endpoint. This framework isolates rogue OAuth 2.0 clients and bad actors, thereby ensuring valid users and applications don't run into rate limit violations.
 
-Client-based rate limiting for the OAuth 2.0 `/authorize` endpoint uses a combination of the Client ID, user's IP address, and Okta device identifier to provide granular isolation for requests made to the OAuth 2.0 `/authorize` endpoint. This framework isolates rogue OAuth 2.0 clients and bad actors, thereby ensuring valid users and applications don't run into rate limit violations.
+The Identity Engine includes client-based rate limiting for multiple APIs that implement the [Interaction Code grant type](/docs/concepts/interaction-code/), which performs a series of interactions and API calls between the user and authorization server.
```

Comment:
The client-based limiting only applies to the "entry points" for those flows where they are bootstrapped. Once they've begun it generally no longer applies.

Reply:
Changed that sentence to: `The Identity Engine includes client-based rate limiting for multiple API entry points that implement...`